src/constants/base.js

```diff
@@ -2,6 +2,7 @@ import { searchResultListColumns } from '@folio/stripes-authority-components';
 
 export const MAIN_PANE_HEIGHT = '609px';
 export const columnWidths = {
+  [searchResultListColumns.LINK]: { min: 45, max: 45 },
   [searchResultListColumns.SELECT]: { min: 30, max: 30 },
   [searchResultListColumns.AUTH_REF_TYPE]: { min: 155, max: 166 },
   [searchResultListColumns.HEADING_REF]: { min: 390, max: 435 },
```

Comment:
We can avoid scrolling by width these values
```
export const columnWidths = {
  [searchResultListColumns.LINK]: { min: 45, max: 45 },
  [searchResultListColumns.SELECT]: { min: 30, max: 30 },
  [searchResultListColumns.AUTH_REF_TYPE]: { min: 95, max: 95 },
  [searchResultListColumns.HEADING_REF]: { min: 410, max: 410 },
  [searchResultListColumns.HEADING_TYPE]: { min: 135, max: 135 },
};
```

Reply:
I think it's easier to read when there's some free space between column content. Otherwise it looks too busy